gateway_test.go

```diff
@@ -129,6 +129,11 @@ func TestMain(m *testing.M) {
 	config.EnableJSVM = true
 	config.Monitor.EnableTriggerMonitors = true
 	config.AnalyticsConfig.NormaliseUrls.Enabled = true
+	config.SlaveOptions.CallTimeout = 30
+	config.SlaveOptions.PingTimeout = 60
+	GlobalRPCPingTimeout = time.Second * time.Duration(config.SlaveOptions.PingTimeout)
+	GlobalRPCCallTimeout = time.Second * time.Duration(config.SlaveOptions.CallTimeout)
```

Comment:
Because the tests would fail without them - it's explained in its commit message. To simplify the logic of what runs and what doesn't regarding the config code in `go test`, now `loadConfig` isn't ran at all. And the last thing it does is set these up.